include/follow.php

```diff
@@ -82,6 +82,11 @@ function new_contact($uid,$url,$interactive = false) {
 		return $result;
 	}
 
+	if (blocked_url($url)) {
+		$result['message'] = t('Blocked domain');
```

Comment:
As the block has a reason associated per domain, maybe we could display it alongside the general note, about the domain being blocked?